XRTK-Core/Packages/com.xrtk.core/Services/InputSystem/FocusProvider.cs

```diff
@@ -852,11 +849,15 @@ private static void UpdatePointerRayOnHit(RayStep[] raySteps, RaycastHit physics
         /// <param name="hitResult"></param>
         private void RaycastGraphics(IMixedRealityPointer pointer, PointerEventData graphicEventData, LayerMask[] prioritizedLayerMasks, PointerHitResult hitResult)
         {
-            Debug.Assert(UIRaycastCamera != null, "Missing UIRaycastCamera!");
+            if (UIRaycastCamera == null)
```

Comment:
I thought one purpose of this PR was to remove the UIRaycastCamera?  So is this error needed anymore?

Reply:
No, this was not the purpose of the PR.